src/org/rascalmpl/util/maven/Artifact.java

```diff
@@ -247,17 +252,6 @@ private static void calculateClassPath(Scope forScope, Queue<ResolveState> resol
                 }
                 alreadyResolved.add(key);
 
-                if (d.getScope() == Scope.PROVIDED) {
-                    // current maven behavior seems to be:
-                    // - do not download provided dependencies
-                    // - if a provided dependency is present in the maven repository it's considered "provided"
```

Comment:
We've lost the behavior that provided jars are not downloaded. so they're only resolved if they are already in the local maven repo, are not downloaded from maven central.